src/components/VErrorSection/VNoResults.vue

```diff
@@ -1,35 +1,56 @@
 <template>
   <div class="no-results text-center md:text-left">
-    <h1 class="break-words text-4xl md:text-6xl">
-      {{ $t("no-results.heading", { query: query.q }) }}
+    <h1 class="heading-4 md:heading-2 break-words">
+      {{ $t("no-results.heading", { query: searchTerm }) }}
     </h1>
-    <h2
-      class="mt-10 text-base font-normal md:mt-16 md:text-3xl md:font-semibold"
-    >
+    <h2 class="description-regular md:heading-5 mt-10 md:mt-16">
       {{ $t("no-results.alternatives") }}
     </h2>
-    <VExternalSourceList class="mt-4 md:mt-6" :type="type" :query="query" />
+
+    <div class="mt-4 flex flex-col flex-wrap gap-4 gap-2 md:mt-6 md:flex-row">
```

Comment:
Similar to my previous comment, I suggest replacing spacing `4` and `6` with `10` (40px) to give more space between headings and buttons list in all breakpoints.